src/slacko.ml

```diff
@@ -335,6 +335,64 @@ type file_obj = {
   num_stars: int option [@default None];
 } [@@deriving of_yojson { strict = false }]
 
+type field_obj = {
+  title: string option [@default None];
+  value: string option [@default None];
+  short: bool option [@default None];
+} [@@deriving yojson { strict = false }]
+
+let field ?title ?value ?short () = {
+  title;
+  value;
```

Comment:
Does it make sense to construct a `field_obj` without a value? Making the `value` mandatory would allow to avoid the `()` argument at maybe no adverse cost to the utility of the API.